packages/drivers/routerlicious-driver/src/restWrapper.ts

```diff
@@ -119,18 +124,28 @@ export class RouterliciousRestWrapper extends RestWrapper {
             "Authorization": this.authorizationHeader!,
```

Comment:
nit: the purpose of authorizationHeader property is now redundant thanks to `this.token`. I think you can remove all the `this.authorizationHeader` lines, and simply replace this 1 line with `"Authorization": this.getAuthorizationHeader(this.token),`